apsuite/orbcorr/orbit_correction.py

```diff
@@ -103,7 +103,7 @@ def get_inverse_matrix(self, jacobian_matrix, full=False):
             return ismat
 
     def correct_orbit(self, jacobian_matrix=None, goal_orbit=None,
-                      update_jac=False):
+                      update_jac=False, factor=1):
```

Comment:
Could you add some description in this method docstring about these arguments?

Reply:
I transformed these arguments into attributes of CorrParams, now there is a brief comment for them.